packages/react-native-platform-override/src/PackageUtils.ts

```diff
@@ -23,9 +23,13 @@ export async function getInstalledRNVersion(): Promise<string> {
 }
 
 /**
- * Return an object representing the package.json of this package
+ * Return an object representing the package.json of our current package
  */
 export function getNpmPackage(): any {
+  if (!require.main) {
```

Comment:
Needed with strict since `require.main` can be null sometimes according to typings? Not sure if that's ever true running in Node though.